metametamodel/builtins.json

```diff
@@ -67,6 +67,7 @@
           "targets": []
         }
       ],
+      "annotations": [],
```

Comment:
I am afraid I will need to manually fix a ton of files in the lioncore-java repo :(

Reply:
I wonder if we should revisit the decision of necessarily having to write empty lists

Reply:
Some tests relies on JSON files in the resources and I would need to update those files. Those files have been obtained from various sources, and I do not have the code to generate all of them. Previously performing changes on those files have been quite time consuming

Reply:
It would be interesting to see how we could generate those files after all.

Reply:
I am referring to the files that I used in tests of the Java implementation, like this one: https://github.com/LIonWeb-org/lioncore-java/blob/master/core/src/test/resources/serialization/TestLang-language.json
Given they are files used in tests, I do not think I should generate them